apps/staking/src/components/WalletTester/index.tsx

```diff
@@ -205,28 +206,28 @@ const getHasAlreadyTested = async (
   return { hasTested: receipt !== null };
 };
 
-const testWallet = async (connection: Connection, wallet: AnchorWallet) => {
+const testWallet = async (
+  connection: Connection,
+  wallet: PythStakingWallet,
+) => {
   const walletTester = new Program(
     WalletTesterIDL as Idl,
     new AnchorProvider(connection, wallet),
   );
   const testMethod = walletTester.methods.test;
   if (testMethod) {
-    await sendTransactions(
-      await TransactionBuilder.batchIntoVersionedTransactions(
-        wallet.publicKey,
-        connection,
-        [
-          {
-            instruction: await testMethod().instruction(),
-            signers: [],
-          },
-        ],
-        {},
-      ),
```

Comment:
you can still use the transaction builder and use it in `wallet.sendTransaction` instead of manually creating it.